### ISSUE TITLE
AttributeError: 'InputDevice' object has no attribute 'absinfo'

### ISSUE THREAD

**Issue by caiocvsilva** [Wazzaps/fingerpaint#7]
When I try to run fingerpaint, I get this error:

```
Using touchpad: XXXXXXX Touchpad
Traceback (most recent call last):
  File "/home/XXX/.local/bin/fingerpaint", line 8, in <module>
    sys.exit(cli())
  File "/home/XXX/.local/lib/python3.9/site-packages/fingerpaint/fingerpaint.py", line 288, in cli
    main(args)
  File "/home/XXX/.local/lib/python3.9/site-packages/fingerpaint/fingerpaint.py", line 165, in main
    x_absinfo = touchpad.absinfo(evdev.ecodes.ABS_X)
AttributeError: 'InputDevice' object has no attribute 'absinfo'

```
I am running Fedora 34, with GNOME DE. (I changed some personal information for XXX).

**Comment by jhossbach** [comment]
Looks like python-evdev needs to be at least v1.3.0, Fedoras stable version uses v1.1.2 (which just turned 3 years yesterday).

**Comment by AlfonsoGangemi** [comment]
I have the same error message with Ubuntu_18.04.

In addition when install

> $ python3 -m pip install --upgrade --user fingerpaint
/usr/lib/python3/dist-packages/secretstorage/dhcrypto.py:15: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
/usr/lib/python3/dist-packages/secretstorage/util.py:19: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
Requirement already satisfied: fingerpaint in ./.local/lib/python3.6/site-packages (1.2)
Requirement already satisfied: pyudev in /usr/lib/python3/dist-packages (from fingerpaint) (0.21.0)
Requirement already satisfied: Pillow in /usr/lib/python3/dist-packages (from fingerpaint) (5.1.0)
Requirement already satisfied: evdev in /usr/lib/python3/dist-packages (from fingerpaint) (0.7.0)

**Comment by jhossbach** [comment]
@Wazzaps you should update the requirement in the setup.py to require `>= v1.3.0` for `python-evdev`

**Comment by Wazzaps** [comment]
on it!

**Comment by Wazzaps** [comment]
Fixed in #9 ([link to release](https://github.com/Wazzaps/fingerpaint/releases/tag/v1.2.1)), sorry for the late reply!
